SharperUniverse.Tests/PersistenceTests.cs

```diff
@@ -82,18 +82,15 @@ public async Task NoExcessMemoryUsage()
 			}
 
 			var provider =
-				new LiteDbProvider(new List<ISharperSystem> {fooSystem, barSystem}, runner)
-				{
-					ConnectionString = $"{Path.GetTempPath()}/SU.db"
-				};
+				new LiteDbProvider(new List<ISharperSystem> {fooSystem, barSystem}, runner);
 
 
 			GC.Collect();
```

Comment:
uhhhh context please?

Reply:
I decided to make the connection string for litedb defaulted - so it's no longer needed to be given in an object initializer.

Reply:
I'm referring to the GC.Collect call.

Reply:
Ah - so I call Collect() - save the amount of memory currently used - generate a bunch of "components" which are then saved, collect again, and then compare memory. The reason for the second collection is to ensure there is actual garbage - and not unused references. If this makes sense?